linkerd/tracing/Cargo.toml

```diff
@@ -14,6 +14,7 @@ ansi = ["tracing-subscriber/ansi"]
 linkerd-error = { path = "../error" }
 tokio = { version = "1", features = ["time"] }
 tracing = "0.1.29"
+tracing-appender = "0.2"
```

Comment:
Is this needed? We removed it from the prior branch

Reply:
ah, i think that snuck in while merging, i'll remove it!